### PR TITLE
NF: No "new Object[]" in queries

### PR DESCRIPTION
**Pull request by Arthur-Milchior**
This PR is supposed to follow https://github.com/ankidroid/Anki-Android/pull/6697 and is a matter of taste. Only the last commit does not appear in the other PR.

Adding `new Object[]{` after each query decreases readability as far as I'm concerned, so I prefer to remove it. 

Testing on my merge branch

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/libanki/sched/Sched.java [hunk 1]
```diff
@@ -634,12 +634,12 @@ private int _lrnForDeck(long did) {
                     "SELECT sum(left / 1000) FROM (SELECT left FROM cards WHERE did = ?"
                             + " AND queue = " + Consts.QUEUE_TYPE_LRN + " AND due < ?"
                             + " LIMIT ?)",
-                    new Object[] {did, (Utils.intTime() + mCol.getConf().getInt("collapseTime")), mReportLimit});
+                    did, (Utils.intTime() + mCol.getConf().getInt("collapseTime")), mReportLimit);
```
Comment (david-allison-1) [inline AnkiDroid/src/main/java/com/ichi2/libanki/sched/Sched.java:637]: From my perspective, `new Object[] {}` makes sense when you want to differentiate between args and the params.

Here in particular, I feel it's harder to reason about.

I don't have a massive opinion on this, but borderline "no"

**Comment by mikehardy** [comment]
Interesting. I am actually irritated by that `new Object[]{}` syntax, it's an eyesore for me when I'm reading, I like this change. I don't feel really strongly about it either but I'm weakly for it, and Arthur clearly likes it or he wouldn't have taken the time. 

I'd merge this but it obviously needs conflicts and compiling sorted out.

**Comment by david-allison-1** [comment]
> Interesting. I am actually irritated by that `new Object[]{}` syntax, it's an eyesore for me when I'm reading, I like this change. I don't feel really strongly about it either but I'm weakly for it, and Arthur clearly likes it or he wouldn't have taken the time.
> 
> I'd merge this but it obviously needs conflicts and compiling sorted out.

Sounds good to me

**Comment by Arthur-Milchior** [comment]
>  Arthur clearly likes it or he wouldn't have taken the time. 

I don't like it really strongly. It was just simple to do and worth a try. I was not sure there was a good reason to create array in the first place

**Comment by mikehardy** [comment]
I may like it the most strongly then, ha! I'd like to see it merge, so I'll mark it for such post CI

**Comment by david-allison-1** [comment]
Can we wait until the branch that this is based off is merged? https://github.com/ankidroid/Anki-Android/pull/6697

**Comment by mikehardy** [comment]
You're right, then this will need a rebase I think so marked waiting for feedback as we process through these - thanks

**Comment by Arthur-Milchior** [comment]
Rebased

**Comment by mikehardy** [comment]
Last rebase needed here as the dependent PR just went in

**Comment by Arthur-Milchior** [comment]
Rebased

**Pull request merged**